sunpy/map/sources/rhessi.py

```diff
@@ -46,12 +44,12 @@ def __init__(self, data, header, **kwargs):
         # Fix some broken/misapplied keywords
         if self.meta['ctype1'] == 'arcsec':
             self.meta['cunit1'] = 'arcsec'
-            self.meta['ctype1'] = 'HPLN-TAN'
+            self._fix_and_warn('ctype1', 'HPLN-TAN')
```

Comment:
There is a difference between your error message "we assumed a missing value for you" and "we are fixing a header to make it standard compliant" which is what this is. I am not sure what the best path forward for that is.

In this case, we know what the header means and we are fixing it so that it's WCS spec compliant. I am not convinced this warrants a warning.

Reply:
I sort of threw this together after https://github.com/sunpy/sunpy/issues/4324#issuecomment-651158078 where you said that "in general exposing warnings to the user about invalid FITS is a good thing, if we always hide these people will never learn".

Reply:
Ah, I see what you mean. I don't think in this case we know that ctype1 has to be be `HPLN-TAN`, so I still think a warning should be raised, but I see that the current warning is not the right warning.